ColorSplit.h

```diff
@@ -0,0 +1,11 @@
+#ifndef COLORSPLIT_H_
+#define COLORSPLIT_H_
+typedef enum MajorColor {WHITE, RED, BLACK, YELLOW, VIOLET, numberOfMajorColorstype} MajorColor_type;

```

Comment:
Nice way to mark the number of major colors.
A maintainer may forget to update this file and the one containing the names while adding a color.
Is there any way to make it intuitive?